code/iaas/auth-logic/src/main/java/io/cattle/platform/iaas/api/auth/dao/impl/AuthDaoImpl.java

```diff
@@ -70,6 +69,29 @@ public Account getAdminAccount() {
     }
 
     @Override
+    public List<Account> searchAccounts(String name) {
```

Comment:
This method should be called searchProjects since that is what it is limited to searching.